src/arithmetic.jl

```diff
@@ -1381,6 +1405,21 @@ end
     return nothing
 end
 
+@inline function div_scalar!(c::Taylor1{T}, scalar::NumberNotSeries,
+        a::Taylor1{T}, k::Int) where {T <: NumberNotSeries}
+    if k==0
+        @inbounds c[0] = scalar*constant_term(c) / constant_term(a)
+        return nothing
+    end
+
+    @inbounds c[k] = scalar * c[k]
+    @inbounds for i = 0:k-1
+        c[k] -= c[i] * a[k-i]
+    end
```

Comment:
The following seems to solve the differences you noticed, but you need to test this thoroughly:

```suggestion
    aux = c[k] = scalar * c[k]
    @inbounds zero!(c, k)
    @inbounds for i = 0:k-1
        c[k] -= c[i] * a[k-i]
    end
    c[k] += aux
   ```

Reply:
Thanks! Your suggestion indeed solves the differences while retaining the performance gains.